packages/serializers/md/src/remark-slate/deserialize.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  BlockQuoteNode,
+  CodeBlockNode,
+  defaultNodeTypes,
+  DeserializedNode,
+  HeadingNode,
+  ImageNode,
+  InputNodeTypes,
+  ItalicNode,
+  LinkNode,
+  ListItemNode,
+  ListNode,
+  MdastNode,
+  OptionType,
+  ParagraphNode,
+  TextNode,
+  ThematicBreakNode,
+} from './ast-types';
+import { forceLeafNode } from './forceLeafNode';
+import { persistLeafFormats } from './persistLeafFormats';
+
+export function deserialize<T extends InputNodeTypes>(
+  node: MdastNode,
+  opts?: OptionType<T>
+) {
+  const types = {
+    ...defaultNodeTypes,
+    ...opts?.nodeTypes,
```

Comment:
Now that it's part of Plate, we should use Plate node types as default values using `getPluginType(editor, key)` so we can get rid of this option